### DISCUSSION TITLE
Automatic exposure

### DISCUSSION THREAD

**Discussion by StyXman** [jim-easterbrook/python-gphoto2#149] [General]
It's not clear to me, and testing did not yield any obvious results, so I will ask:

What mode a camera must be so I can change the `f_number` but let the camera compensate with exposure time? I have a Nikon D7200 and when I try to automatically take photos at different `f_number`s, no matter what mode the camera is (P, A, S or M), the exposure time is always the same (whatever was last used in the camera).

**Comment by jim-easterbrook** [comment]
I don't know - it's not something I've tried. With my Canon I'd expect to have to set the dial to `Av` - I don't know if that's the same as your `A` mode. You might reach more people (and maybe some Nikon users) on the gphoto2 user mailing list.

**Reply by lxne** [reply]
`A` is definitely the same as `Av` and that is what you need to go for.
It is 'Aperture Mode' where you can set the aperture manually and ISO and/or shutter speed will be set by the camera.
